packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -162,48 +162,38 @@ const styles: ComponentStyles<SiteManagedClasses, DevSiteDesignSystem> = {
     site_infoBarConfiguration: {
         display: "flex",
         alignItems: "center",
-        padding: toPx(4),
+        padding: "2px",
     },
     site_infoBarConfiguration_base: {
         position: "relative",
-        "&::before, &::after": {
+        "&::before": {
+            right: "8px",
+            top: "9px",
             content: "''",
             position: "absolute",
-            top: toPx(11),
-            zIndex: "1",
-            borderRadius: toPx(2),
-            width: toPx(1),
-            height: toPx(10),
-            background: (config: DevSiteDesignSystem): string => {
-                return config.foreground300 || devSiteDesignSystemDefaults.foreground300;
+            borderTop: (config: DevSiteDesignSystem): string => {
+                return `3px solid ${config.foreground300 ||
+                    devSiteDesignSystemDefaults.foreground300}`;
             },
-        },
-        "&::before": {
-            right: toPx(15),
-            transform: "rotate(45deg)",
-        },
-        "&::after": {
-            right: toPx(22),
-            transform: "rotate(-45deg)",
+            borderLeft: "3px solid transparent",
+            borderRight: "3px solid transparent",
         },
     },
     site_infoBarConfiguration_theme: {
         marginRight: toPx(4),
     },
     site_infoBarConfiguration_input: {
-        lineHeight: toPx(16),
-        fontSize: toPx(14),
         backgroundColor: (config: DevSiteDesignSystem): string => {
             return config.background350 || devSiteDesignSystemDefaults.background350;
         },
         color: (config: DevSiteDesignSystem): string => {
             return config.foreground300 || devSiteDesignSystemDefaults.foreground300;
         },
+        fontFamily: "inherit",
+        fontSize: "inherit",
         borderRadius: toPx(2),
         appearance: "none",
-        padding: localizeSpacing(Direction.ltr)(
-            `${toPx(8)} ${toPx(36)} ${toPx(8)} ${toPx(10)}`
-        ),
+        padding: localizeSpacing(Direction.ltr)("2px 36px 3px 10px"),
```

Comment:
I noticed in some places we are localizing and in others we are not...

Reply:
If there is an issue we can address it in another PR. As long as none of them were removed.